src/corehost/cli/deps_format.cpp

```diff
@@ -134,16 +134,18 @@ bool deps_json_t::perform_rid_fallback(rid_specific_assets_t* portable_assets, c
         {
             if (rid_fallback_graph.count(host_rid) == 0)
             {
-                trace::error(_X("Did not find fallback rids for package %s for the host rid %s"), package.first.c_str(), host_rid.c_str());
-                return false;
+                trace::warning(_X("Did not find fallback rids for package %s for the host rid %s"), package.first.c_str(), host_rid.c_str());
             }
-            const auto& fallback_rids = rid_fallback_graph.find(host_rid)->second;
-            auto iter = std::find_if(fallback_rids.begin(), fallback_rids.end(), [&package](const pal::string_t& rid) {
-                return package.second.rid_assets.count(rid);
-            });
-            if (iter != fallback_rids.end())
+            else
             {
-                matched_rid = *iter;
+                const auto& fallback_rids = rid_fallback_graph.find(host_rid)->second;
```

Comment:
@gkhanna79, this error can occur only if rid graph is totally absent or someone patched hostpolicy built for a higher rid into a lower FX.

This means we cannot find the host rid in the netcore.app fallback graph in the FX deps json. Since it is not an error to skip a package (for loop not matching), it is also not an error to start the fallback for the package.